.github/scripts/github_utils.py

```diff
@@ -55,6 +55,7 @@ def gh_fetch_json(
     url: str,
     params: Optional[Dict[str, Any]] = None,
     data: Optional[Dict[str, Any]] = None,
+    method: Optional[str] = None,
```

Comment:
There is nothing wrong with this change, but technically speaking it's no longer needed